packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.scss

```diff
@@ -67,6 +67,10 @@
             margin-bottom: 2.4rem;
         }
 
+        &--disabled {
+            opacity: 0.5;
+        }
+
```

Comment:
`disabled` prop should handle the disable style for Dropdown component, why do we need this change?

Reply:
ah its cause in the design, the input was less visible and even though disabled is true, the input is still quite visible... below in the first pic is if i dont use the `--disabled` styles, and the second one, is the design for the transfer modal

<img width="483" alt="Screenshot 2023-10-05 at 2 36 53 PM" src="https://github.com/suisin-deriv/deriv-app/assets/103412909/b41a5cc0-d1d3-4481-a634-ff8d4d15620c">
<img width="492" alt="Screenshot 2023-10-05 at 2 36 44 PM" src="https://github.com/suisin-deriv/deriv-app/assets/103412909/1d3138cd-61a7-4874-b13d-d80d4debb2c4">

Reply:
Got the reason for using `--disabled` but from a design system perspective, if a disabled `Dropdown` should look like the 2nd image, shouldn't all disabled `Dropdown`s in our application look the same?

Reply:
right? i dont think all the dropdowns for disabled look the same throughout deriv app, like for p2p the entire dropdown when disabled doesn't look like that... we should discuss this with the design team to make everything consistent